### PR TITLE
Added woofi optimism dex 

### PR DESCRIPTION
**Pull request by ARDev097**
Created spell for adding trades data of woofi dex on optimism.

### CODE REVIEW

File: models/openocean/optimism/openocean_optimism_trades.sql [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{ config(
-    
+    tags = ['dunesql'],
```
Comment (Hosuke) [inline models/openocean/optimism/openocean_optimism_trades.sql:2]: By the way, we can remove `dunesql` tags now, as all existing sqls are dune sql.

Comment (ARDev097) [reply models/openocean/optimism/openocean_optimism_trades.sql:2]: Hello @Hosuke ,

I've implemented the changes you suggested and removed the dunesql tag. Please let me know if there are any additional changes required to the spell.

**Comment by Hosuke** [comment]
I will try to use the macro approach to implement `woofi_optimism_trades` in this PR as well.

**Comment by Hosuke** [comment]
I will try to add seed and test in this PR to keep consistent as in #4774 .

File: seeds/dex/trades/dex_trades_seed.csv [hunk 2]
```diff
@@ -473,14 +473,21 @@ arbitrum,camelot,3,2023-10-15,0x810b6259d2171ca029a3ddb86dfbf682a930644342408219
...
+optimism,openocean,2,2023-05-30,0xc97e676905d6d2763fa55ac9aac6d6d85f6597901cf34711d6b65981b0cb0f75,7,0x7f5c764cbc14f9669b88837ca1490cca17c31607,3.193448,0x4200000000000000000000000000000000000006,0.00167453389173889
+optimism,openocean,2,2023-11-07,0xde2d7e6f02cfce8eb8a1bb752ce956cfcf81b7bf8380490bcdfeab9694b4e5ad,15,0x8700daec35af8ff88c16bdf0418774cb3d7599b4,2.9738578135579843,0x4200000000000000000000000000000000000006,0.003958725899153863
+optimism,openocean,2,2023-07-31,0x92188637370c0bf3595ae4bef248db90f43167483320fc9b21adb016cfb84e76,81,0x8b21e9b7daf2c4325bf3d18c1beb79a347fe902a,1438.894027919516,0x4200000000000000000000000000000000000042,14.86875
 base,maverick,1,2023-10-24,0xf76389f7730beb6a51a3e1881b0a9139fe8ec07d8403061fd52d5aaf1357ad00,12,0x50c5725949a6f0c72e6c4a641f24049a917db0cb,2955.014794642196128309,0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca,2954.059434
+zksync,maverick,1,2023-10-23,0xbe2700aac1174728513863a4f42d37aa9ac7a7d73389fa00a5abcfbd3570d5a6,4,0x2039bb4116B4EFc145Ec4f0e2eA75012D6C0f181,,0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4,
```
Comment (Hosuke) [inline seeds/dex/trades/dex_trades_seed.csv:487]: By the way @ARDev097, why do you add this `zksync,maverick,1` seed record here？

Comment (ARDev097) [reply seeds/dex/trades/dex_trades_seed.csv:487]: Sorry @Hosuke for it, but there is some conflict issue, so when i resolved it was added by mistakenly

**Comment by ARDev097** [comment]
Hello @Hosuke , Please clear the error by running once again. If I'll do that then due to conflict something will go wrong.

File: models/openocean/optimism/openocean_optimism_trades.sql (hunk 1 above)
Comment (Hosuke) [inline models/openocean/optimism/openocean_optimism_trades.sql:2]:
```suggestion
```

File: seeds/_sector/dex/_schema.yml [hunk 3]
```diff
@@ -1335,6 +1335,36 @@ seeds:
...
+        version: varchar
+        tx_hash: varbinary
+        evt_index: uint256
+        block_number: uint256
+        token_bought_address: varbinary
+        token_sold_address: varbinary
+        token_bought_amount_raw: uint256
+        token_sold_amount_raw: uint256
+        block_date: timestamp
+
```
Comment (Hosuke) [inline seeds/_sector/dex/_schema.yml:1367]:
```suggestion
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅ for woofi_optimism.trades
Thank you @ARDev097

**Review by jeff-dude — APPROVED** [review]
thank you both!

**Pull request merged**